### PR TITLE
Set up the Gift and Wallet tables in the DB for the Payment TC2

### PR DESCRIPTION
**Pull request by realswat1**
this is the first step into our tc 2 . I created the wallet and gift models to all the accurate keep of data for all transactions and data recording . The test plan was to get the data tables in database
image

### CODE REVIEW

File: exposee-app/models/wallet.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        allowNull: false,
+    },
+    deductions: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+    },
+    addition: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+    }
```
Comment (Rajeshwari21) [inline exposee-app/models/wallet.js:20]: should we add the id of the user to whom the amount is been sent, to keep a track of it? cc @alfonsocheng

File: exposee-app/models/index.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
 import { Video } from "./video.js";
+import { Gift } from "./gifts.js";
+import { Wallet } from "./wallet.js";
 
 User.hasMany(Video, { as: "videos", foreignKey: "userId" });
 Video.belongsTo(User, { as: "user", foreignKey: "userId" });
+User.hasMany(Gift, { as: "gifts", foreignKey: "userId" });
+Gift.belongsTo(User, { as: "user", foreignKey: "userId" });
+User.hasOne(Wallet, { as: "wallet", foreignKey: "userId" });
+Wallet.belongsTo(User, { as: "user", foreignKey: "userId" });
```
Comment (alfonsocheng) [inline exposee-app/models/index.js:11]: these foreign keys aren't actually tested but you can change them later when it's ready for testing

File: exposee-app/models/wallet.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+export const Wallet = sequelize.define("Wallet", {
+    user_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+    },
+    amount: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+    },
+    deductions: {
```
Comment (alfonsocheng) [inline exposee-app/models/wallet.js:13]: this table could just be the amount without deductions and addition, but I'm OK to leave them in and not use them.

**Review by alfonsocheng — APPROVED** [review]
please update the PR so that the loading spinner code isn't here but otherwise db looks good!

File: exposee-app/models/wallet.js (hunk 1 above)
Comment (realswat1) [reply exposee-app/models/wallet.js:20]: in the gifts , I added receivers and senders id to allow us to track transactions.

Comment (alfonsocheng) [reply exposee-app/models/wallet.js:20]: yeah I see it as part of the gifts table so I think it's good!

**Pull request merged**